src/utils/conversion.js

```diff
@@ -0,0 +1,100 @@
+function convertAction(action, newProperties) {
+  const newAction = {
+    actionType: action.actionType,
+    bookmark: action.selectedBookmark,
+    // Will need to convert field selected by expression
+    field: action.selectedField,
+    // Need to convert expression for variable
+    variable: action.variable,
+    value: action.value,
+    softLock: action.softLock,
+    cId: action.cId,
+  };
+  switch (action.actionType) {
+    case 'clearOther':
+      newAction.actionType = 'clearAllButThis';
+      break;
+    case 'unlockAllAndClearAll':
+      newAction.actionType = 'clearAll';
+      newAction.softLock = true;
+      break;
+    case 'selectField':
+      newAction.actionType = 'selectValues';
+      break;
+    case 'selectAndLockField':
+      newProperties.actions.push({ ...newAction, cId: 'test', actionType: 'selectValues' });
```

Comment:
Almost forgot that I added a hard coded test here. Good that I'm writing some unit tests :)